Gemfile

```diff
@@ -22,6 +23,14 @@ group :development do
   gem 'spring-watcher-listen'
 end
 
+group :test do
+  gem 'rails-controller-testing'
+  gem 'minitest'
+  gem 'minitest-reporters'
+  gem 'guard'
```

Comment:
I thought you decided not to use guard. As a rule of thumb, minimize the the number of external dependencies. Never add a gem for reasons like "why not?"

Reply:
We talked about it but I forgot that it was already added. Will remove.